Octokit/Http/ApiInfoParser.cs

```diff
@@ -8,7 +8,7 @@ namespace Octokit.Internal
     internal static class ApiInfoParser
     {
         const RegexOptions regexOptions =
-#if NETFX_CORE
+#if !HAS_REGEX_COMPILED_OPTIONS
```

Comment:
this block could be written the same as [here](https://github.com/octokit/octokit.net/pull/1503/files#diff-9b75446537d00a96e613eefafd903662R126) for consistency